bin/tornadovm-installer.cmd

```diff
@@ -0,0 +1,233 @@
+@echo off & setlocal
```

Comment:
The script worked out of the box for me and I appreciate you keep the same logic as we have in the Python script installer. As I mentioned in the general discussion, is there anything specific, or do you think in the future we could migrate everything to the same Python installer?

Reply:
There seems to me nothing specific in the Python script installer. It should therefore be possible to migrate the CMD script installer to Python, to make it work on Windows as well as on Linux.